cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -1172,8 +1172,11 @@
     <name>log.publish.partition.key</name>
     <value>program</value>
     <description>
-      Publish logs from a program or an app to same partition. Valid values are "application" and "program".
-      Changes to this property requires restart of all cdap applications.
+      Publish logs from a program or an application to the same partition, depending on
+      the key value. Valid values are "application" or "program". If set to "application",
```

Comment:
What do mean by "depending on the key value"? Maybe we don't need to say that? 
`Publish logs from a program or an application to the same partition. Valid values are "application" or "program".`